src/deidentify/address_to_census_tract.py

```diff
@@ -196,7 +196,9 @@ def process_csv_or_excel(filepath: str, output: str, address_map: dict,
 
     # Drop identifiable address columns
     drop_columns = list(address_map.values())
-    keep_zipcode and drop_columns.remove(address_map['zipcode'])
+
+    if not keep_zipcode:
+        drop_columns.remove(address_map['zipcode'])
```

Comment:
The logic here is was changed, and should be, I believe: ```
if keep_zipcode:
    drop_columns.remove(address_map['zipcode'])
```

since `drop_columns` is a list of the columns to drop.

Reply:
ah yes, thank you!